.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,60 @@
+name: Feature Request
+description: File a feature request
+title: "[Feature]: "
+labels: ["Feature"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this Feature request report!
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Suggest an idea for this project ?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+
+  - type: textarea
+    id: screenshot
+    attributes:
+      label: Please provide an example screenshot
+      placeholder: screenshot
+    validations:
+      required: true
```

Comment:
Screenshots should be optional for both bug and feature request templates.

Reply:
done